transport/grpc/handler.go

```diff
@@ -180,8 +180,16 @@ func (h *handler) handleUnary(
 	err := h.handleUnaryBeforeErrorConversion(ctx, transportRequest, responseWriter, start, handler)
 	err = handlerErrorToGRPCError(err, responseWriter)
 
+	// We need to copy the bytes out so that we do not use the buffer-pool-backed byte slices.
```

Comment:
i don't think this comment makes sense anymore? we can revert this change right?